docs/source/user_guide/grid_methods/99_uncategorized.rst

```diff
@@ -0,0 +1,24 @@
+Uncategorized and Deprecated
```

Comment:
Is this category only for deprecated methods/functions, or also for otherwise uncategorized but not deprecated ones?

Reply:
These are functions that are either deprecated *or* uncategorized. In actuality, it's almost completely uncategorized functions—there are only a few deprecated functions.

Reply:
Probably best to clarify that in the docs, so people don't mistakenly think that any uncategorized function has been deprecated.

Reply:
Also, I would lean toward keeping our binder links (in addition to JupyterHub) so people can try out a notebook without needing a login.

Reply:
I've updated the wording and added a couple sentences clarifying that the functions are *either* uncategorized *or* deprecated.